src/python/WMCore/ReqMgr/DataStructs/Request.py

```diff
@@ -20,6 +20,9 @@
 from WMCore.ReqMgr.DataStructs.RequestStatus import REQUEST_START_STATE
 from WMCore.ReqMgr.DataStructs.RequestError import InvalidSpecParameterValue
 
+#TODO: this need to be update from DB instead of hard coding for more flexibility
+RESULT_FILTER_FOR_DAS = ["RequestName", "OutputDatasets", "InputDatasets", "PrepID"]
```

Comment:
This the data will be returned when filter=DAS is set. @amaltaro, Alan please review what information we should add for this. I have no idea. :smile: